syntaxes/.tmLanguage.json

```diff
@@ -19,28 +18,26 @@
 		"keywords": {
 			"patterns": [{
 				"name": "keyword.control.lpg",
-				"match": "\\b(let|enum|struct|match|case|loop|break|return|interface|impl)\\b"
+				"match": "\\b(let|enum|struct|match|case|loop|break|return|mutable|interface|reference|new|impl|import)\\b"
```

Comment:
mutable, reference und new gibt es nicht.
Es fehlen noch type-of und new-array.